lib/board.rb

```diff
@@ -27,6 +29,11 @@ def clear_mark(move)
     @board_array[position] = move
   end
 
+  def get_opponent_mark(given_mark)
+    opponent = PLAYER_MARKS.reject{|player_mark| player_mark == given_mark}
```

Comment:
Could you please try and rewrite this using: https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-find ?

Reply:
Thanks, will give this a try